src/radical/pilot/context.py

```diff
@@ -11,18 +11,20 @@
 __copyright__ = "Copyright 2013-2014, http://radical.rutgers.edu"
 __license__   = "MIT"
 
-import saga
+try:    import         saga as rs
+except: import radical.saga as rs
+
 
 # ------------------------------------------------------------------------------
 #
-class Context (saga.Context):
+class Context (rs.Context):
 
     # --------------------------------------------------------------------------
     #
     def __init__ (self, ctype, thedict=None) :
 
         # init the saga.Context
-        self._apitype  = 'saga.Context'
+        self._apitype  = 'radical,saga.Context'
```

Comment:
There is a typo here: 
```suggestion
        self._apitype  = 'radical.saga.Context'
```

Reply:
Thanks!